Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_Generic_v2.yml

```diff
@@ -191,27 +218,21 @@ inputs:
       - operator: uniq
   required: false
   description: The IP address to enrich.
+  playbookInputQuery:
 - key: InternalRange
   value:
-    complex:
-      root: inputs.InternalRange
-      transformers:
-      - operator: uniq
+    simple: 172.16.0.0/12,10.0.0.0/8,192.168.0.0/16
   required: false
   description: 'A list of internal IP ranges to check IP addresses against. The list should be provided in CIDR notation, separated by commas. An example of a list of ranges would be: "172.16.0.0/12,10.0.0.0/8,192.168.0.0/16" (without quotes). If a list is not provided, will use default list provided in the IsIPInRanges script (the known IPv4 private address ranges).'
+  playbookInputQuery:
 - key: ResolveIP
   value:
-    complex:
-      root: inputs.ResolveIP
-      transformers:
-        - operator: SetIfEmpty
-          args:
-            applyIfEmpty: { }
-            defaultValue:
-              value:
-                simple: "False"
+    simple: "True"
```

Comment:
why not false? 
- I see that the previous value had a transformer of `SetIfEmpty` with a default of "false".